qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/CreateCustomerView.groovy

```diff
@@ -98,28 +103,25 @@ class CreateCustomerView extends VerticalLayout {
         row2.setSizeFull()
 
         HorizontalLayout row3 = new HorizontalLayout(affiliationComboBox, addressAdditionComboBox)
-        row3.setComponentAlignment(affiliationComboBox, Alignment.BOTTOM_LEFT)
-        row3.setComponentAlignment(addressAdditionComboBox, Alignment.BOTTOM_LEFT)
+        row3.setComponentAlignment(affiliationComboBox, Alignment.TOP_LEFT)
+        row3.setComponentAlignment(addressAdditionComboBox, Alignment.TOP_LEFT)
         row3.setSizeFull()
 
-        HorizontalLayout row4 = new HorizontalLayout(affiliationButton)
-        row4.setComponentAlignment(affiliationButton, Alignment.BOTTOM_LEFT)
-        row4.setSizeFull()
-
-        HorizontalLayout row5 = new HorizontalLayout(submitButton)
+        HorizontalLayout row5 = new HorizontalLayout(affiliationDetails, submitButton)
```

Comment:
```suggestion
        HorizontalLayout row4 = new HorizontalLayout(affiliationDetails, submitButton)
```
You removed a row so I suspect that this should also be renamed

Reply:
actually, that was already not clean code. numbering the var names is as we just see here causing unnecessary overhead.